SlevomatCodingStandard/Helpers/ClassHelper.php

```diff
@@ -18,4 +18,22 @@ public static function getName(\PHP_CodeSniffer_File $codeSnifferFile, int $clas
 		return $tokens[$codeSnifferFile->findNext(T_STRING, $classPointer + 1, $tokens[$classPointer]['scope_opener'])]['content'];
 	}
 
+	public static function getAllNames(\PHP_CodeSniffer_File $codeSnifferFile): array
+	{
+		$getAllClassPointers = function(\PHP_CodeSniffer_File $codeSnifferFile, &$previousClassPointer): \Generator {
```

Comment:
Unfortunatelly, there cannot be int typehint, because it can be either int or false. Because php 7.0 does not have union types in parameters I have extracted lamda function to separate private static method and at least annotated `@param` with union type `int|bool`.